mkdocs.yml

```diff
@@ -207,11 +207,11 @@ nav:
         - 'Using Tor': guides/misc/tor/using-tor.md
         - 'Performance and other issues': guides/misc/tor/performance-issues.md
         - 'Using DNSSEC': guides/misc/tor/dnssec.md
-      - 'Whitelist and Blacklist editing': guides/misc/whitelist-blacklist.md
+      - 'Allowlist and Denylist editing': guides/misc/allowlist-denylist.md
     - 'Web server':
-      - 'NGINX': guides/webserver/nginx.md
-      - 'Caddy': guides/webserver/caddy.md
-      - 'Traefik v1 (not in Docker)': guides/webserver/traefik-nodocker.md
+      #- 'NGINX': guides/webserver/nginx.md
+      #- 'Caddy': guides/webserver/caddy.md
+      #- 'Traefik v1 (not in Docker)': guides/webserver/traefik-nodocker.md
```

Comment:
I think traefik v1 can be removed completely. v3 is released already. Users should upgrade.